magento/magento2
# (bugfix) #30026 Fix depending from multiselects

### Description (*)

Allows depending from a single option in a backend multiselect. See issue for step-by-step


### Fixed Issues (if relevant)

1. Fixes magento/magento2#30026

### Manual testing scenarios (*)

1. Create a module with system.xml
2. Make one field depend on a value being selected in a separate multiselect
3. Select the value and one other value
4. Observe as the depending field becomes/remains visible
5. Unselect the value declared in the depends node
6. Observe as the depending field becomes hidden

### Questions or comments


### Contribution checklist (*)
